src/Tools/dotnet-dump/ExtensionCommands/TaskStateCommand.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Diagnostics.Repl;
+
+namespace Microsoft.Diagnostic.Tools.Dump.ExtensionCommands
+{
+    [Command(Name = "taskstate", AliasExpansion = "TaskState", Help = "Display a Task state in a human readable format.")]
```

Comment:
You don't need the AliasExpansion property. It is a special case for the native SOS commands.

Reply:
ok